boot/Lexer.x

```diff
@@ -13,11 +13,24 @@
 #ifdef CABAL_PARSEC_DEBUG
 {-# LANGUAGE PatternGuards #-}
 #endif
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
 module Distribution.Parsec.Lexer
   (ltest, lexToken, Token(..), LToken(..)
   ,bol_section, in_section, in_field_layout, in_field_braces
   ,mkLexState) where
 
+-- [Note: boostrapping parsec parser]
```

Comment:
Won't do. If something, then `alex` should add notes for files generated with it. I'm heavily :-1: of doing something manual / only for Cabal at this point.